### PR TITLE
This commit fixes #53.  Adding meteor checks to them with a global ch…

### LINKED ISSUE #53: Add `check`s to all our meteor methods
Make sure all of our Meteor Methods have checks for the parameters at the beginning of the method

### PR DESCRIPTION
**Pull request by ZachStoltz**
Fixing the Meteor.methods to use the check function for the defined function params.

### CODE REVIEW

File: server/methods/initiative.js [hunk 1]
```diff
@@ -1,7 +1,32 @@
...
+    active: Boolean,
+    category: String,
+    categorySlug: String,
+    createdAt: Number,
+    createdBy: String,
+    imageUrl: String,
+    slug: String,
+    summary: String,
+    title: String,
+    comments: Match.Optional ([String]),
```
Comment (jeffshaver) [inline server/methods/initiative.js]: `Match` is a meteor global. It needs to be defined with the rest of them in .eslintrc. Also, no space after `Match.Optional`

File: server/methods/initiative.js [hunk 2]
```diff
@@ -1,7 +1,32 @@
+/*
+ * Global function to help reduce boilerplate code for Meteor
+ * checks on the initiative object
+ * @param initiative
+ */
+checkInitiative = function checkInitiative(initiative) {
+  check (initiative, {
```
Comment (jeffshaver) [inline server/methods/initiative.js]: no space after `check`

File: server/methods/initiative.js [hunk 3]
```diff
@@ -1,7 +1,32 @@
...
+    category: String,
+    categorySlug: String,
+    createdAt: Number,
+    createdBy: String,
+    imageUrl: String,
+    slug: String,
+    summary: String,
+    title: String,
+    comments: Match.Optional ([String]),
+    usersVoted: Match.Optional ([String]),
```
Comment (jeffshaver) [inline server/methods/initiative.js]: `Match` is a meteor global. It needs to be defined with the rest of them in .eslintrc. Also, no space after `Match.Optional`

Comment (ZachStoltz) [inline server/methods/initiative.js]: Making changes now!  Thanks for catching this!

**Comment by jeffshaver** [comment]
change line 107 of .eslintrc:

```
"new-cap": 1,
```

to be 

```
"new-cap": [1, {newIsCap: true, capIsNew: false}],
```

This will fix the errors in regards to Match.Optional not being a constructor.

**Comment by iamlasse** [comment]
Great job guys and good to see the flow starting again.

**Pull request merged**